examples/express-ts/package.json

```diff
@@ -8,26 +8,30 @@
     "compile": "tsc",
     "start": "tsc && node lib/app.js"
   },
-  "author": "Sami Koskimäki and Matthew McEachen",
+  "author": "Sami Koskimäki",
+  "contributors": [
+    "Matthew McEachen <https://github.com/mceachen>"
+  ],
   "license": "MIT",
+  "repository": "n/a",
   "dependencies": {
-    "body-parser": "^1.15.1",
-    "express": "^4.13.4",
-    "knex": "^0.10.0",
-    "lodash": "^4.12.0",
+    "body-parser": "^1.15.2",
+    "express": "^4.14.0",
+    "knex": ">= 0.8.0 <= 0.12.5",
+    "lodash": "^4.17.2",
     "morgan": "^1.7.0",
-    "objection": "^0.6.1",
-    "sqlite3": "^3.1.3"
+    "objection": "../..",
```

Comment:
The local version (head) may not be compatible with the example project. Unless you are willing to keep it up to date with the most recent updates all the time, this should be reverted